willow/registry.py

```diff
@@ -34,6 +38,7 @@ def __init__(self):
         self._registered_operations = defaultdict(dict)
         self._registered_converters = {}
         self._registered_converter_costs = {}
+        self._registered_optimizers: List["OptimizerBase"] = set()
```

Comment:
Drive-by comment: the value is still `set()`

```suggestion
        self._registered_optimizers: List["OptimizerBase"] = []
```

Reply:
🤦🏼 I thought I updated that 🏃‍♂️